Directory.Build.props

```diff
@@ -103,7 +103,6 @@
          numbers of the shared frameworks. -->
     <MicrosoftWindowsDesktopAppRuntimePackageVersion>$(MicrosoftNETCoreAppRefPackageVersion)</MicrosoftWindowsDesktopAppRuntimePackageVersion>
     <MicrosoftWindowsDesktopAppRefPackageVersion>$(MicrosoftNETCoreAppRefPackageVersion)</MicrosoftWindowsDesktopAppRefPackageVersion>
-    <MicrosoftNETSdkWindowsDesktopPackageVersion>$(MicrosoftNETCorePlatformsPackageVersion)</MicrosoftNETSdkWindowsDesktopPackageVersion>
```

Comment:
@mmitche this probably shouldn't have been removed

Reply:
Nope that is correct....this is https://github.com/dotnet/dotnet/issues/1512

I do need someone to take over that issue if you have some bandwidth.